File: delivery-app/app/src/main/java/com/deliveryperson/data/model/DeliveryPerson.kt
package com.deliveryperson.data.model

data class DeliveryPerson(
    val id: String,
    val name: String,
    val phone: String,
    val email: String,
    val vehicle_type: String,
    val is_available: Boolean,
    val current_latitude: Double?,
    val current_longitude: Double?,
    val created_at: String
)

File: backend/models/delivery_person.py
"""
Delivery Person model - Pure data structure
"""
from dataclasses import dataclass
from typing import Optional

@dataclass
class DeliveryPerson:
    """Delivery person model"""
    id: str
    name: str
    phone: str
    email: str
    password_hash: str  # Hashed password
    vehicle_type: str = "bike"
    license_number: Optional[str] = None
    is_available: bool = False
    current_latitude: Optional[float] = None
    current_longitude: Optional[float] = None
